packages/react/cypress/component/auto/form/AutoFormInputLabels.cy.tsx

```diff
@@ -11,9 +11,13 @@ import { api } from "../../../support/api.js";
 import { describeForEachAutoAdapter } from "../../../support/auto.js";
 import { SUITE_NAMES } from "../../../support/constants.js";
 
-const ShadcnAutoInput = makeShadcnAutoInput({ Input, Label, Checkbox, Button });
+function useShadcnAutoInput() {
+  return React.useMemo(() => makeShadcnAutoInput({ Input, Label, Checkbox, Button }), []);
+}
```

Comment:
I don't think this is super realistic -- users aren't gonna wrap their factories in a memo, you know? I think they are just gonna have one big file that defines all their autocomponents once as module scope variables, and then they're gonna import from there. Does it still work if you leave it as it was?

Reply:
We don't need the hook and memoization. I removed it. Thanks